src/ResourceManager/DataFactories/Commands.DataFactories.Test/ScenarioTests/DataFactoryGatewayTests.cs

```diff
@@ -37,6 +37,12 @@ public void TestCreateDataFactoryGateway()
             RunPowerShellTest("Test-DataFactoryGateway");
         }
 
+        [Fact]
```

Comment:
@nicolaszhang this test is failing on the on-demand run because the json file was not added to the test csproj file, so it is not being copied over to the output directory. If you follow the pattern that you do [for other json files](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/DataFactories/Commands.DataFactories.Test/Commands.DataFactories.Test.csproj#L291-L293) and add the few lines to the csproj, then the build should be fine.

Also, would you mind making this test a check-in test so it runs during the regular build and not just on-demand? The test below this one has the traits that you need to make it a check-in test

```cs
[Trait(Category.AcceptanceType, Category.CheckIn)]
```